src/components/ClubCard.js

```diff
@@ -0,0 +1,149 @@
+import React, { Component } from "react";
+import FA from "@fortawesome/react-fontawesome";
+import PropTypes from "prop-types";
+import { css } from "styled-components";
+import { Box, Card, Flex, Heading, Link, Text } from "@hackclub/design-system";
+import axios from "axios";
+import geolib from "geolib";
+
+const Base = Box.extend`
+    display: flex;
+    padding: ${props => props.theme.space[2]}px;
+    text-align: initial;
+    width: 100%;
+    ${props => props.theme.mediaQueries.sm} {
+        width: 50%;
+    }
+    ${props => props.theme.mediaQueries.md} {
+        width: 33.3333%;
+    }
+    ${props => props.theme.mediaQueries.lg} {
+        width: 25%;
+    }
+`;
+
+const Inner = Card.withComponent(Flex).extend.attrs({
+    bg: "snow",
+    boxShadowSize: "sm",
+    flexDirection: "column"
+})`
+    border-radius: ${props => props.theme.radius};
+    position: relative;
+    transition: 0.125s box-shadow ease-in, 0.125s transform ease-in;
+    width: 100%;
+    &:hover {
+        box-shadow: ${props => props.theme.boxShadows[2]};
+        transform: scale(1.02);
+    }
+`;
+
+const Distance = Text.span.extend.attrs({
+    color: "white",
+    p: 2
+})`
+    background: rgba(0, 0, 0, 0.25);
+    border-radius: ${props => props.theme.radius} 0 ${props => props.theme.radius} 0;
+    position: absolute;
+    text-shadow: rgba(0, 0, 0, 0.32) 0px 1px 4px;
+    z-index: 1;
+`;
+
+const Photo = Box.withComponent("figure").extend.attrs({
```

Comment:
use the `BackgroundImage` DS component, note that the `image` prop is used instead of `src`

Reply:
Problem:
```
Over 200 classes were generated for component Styled(Base). 
Consider using the attrs method, together with a style object for frequently changed styles.
```

Reply:
I think it's how the `image` prop is used in DS: https://github.com/hackclub/design-system/blob/master/src/BackgroundImage.js#L5-L16